addons/pos_loyalty/static/tests/tours/GiftCardProgramTours.js

```diff
@@ -36,6 +36,23 @@ registry.category("web_tour.tours").add("GiftCardProgramCreateSetTour2", {
 });
 //#endregion
 
+//#region GiftCardProgramPriceTour
+registry.category("web_tour.tours").add("GiftCardProgramPriceNoTaxTour", {
+    test: true,
+    url: "/pos/web",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+
+            // Use gift card
+            ProductScreen.addOrderline("Magnetic Board", "1", "1.98", "1.98"),
+            PosLoyalty.enterCode("043123456"),
+            Dialog.confirm(),
```

Comment:
You can add a step to test that the gift card is in the order. Something like ´selectedOrderlineHas´ should do the trick